services/github/github.tester.js

```diff
@@ -74,7 +74,7 @@ t.create('License - API rate limit exceeded')
       message: "API rate limit exceeded for 123.123.123.123. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)",
       documentation_url: "https://developer.github.com/v3/#rate-limiting"
   }))
-  .expectJSON({ name: 'license', value: 'inaccessible', colorB: colorsB.lightgrey });
+  .expectJSON({ name: 'license', value: 'invalid', colorB: colorsB.lightgrey });
```

Comment:
This change is due to the default _error-helper.js_ handling. Wouldn't `inaccessible` be more appropriate for 403 responses everywhere?

Reply:
It could be worth updating the error-helper to something similar to this:
```javascript
defaultErrorMessages = {
  404: 'not found',
  418: "I'm a teapot",
};

const checkErrorResponse = function(badgeData, err, res, errorMessage = 'not found') {
  if (typeof errorMessage === 'string')
    errorMessage = {...defaultErrorMessages, ...{404: errorMessage}}
  else
    errorMessage = {...defaultErrorMessages, ...errorMessage};
  if (err != null) {
    badgeData.text[1] = 'inaccessible';
    badgeData.colorscheme = 'red';
    return true;
  } else if (errorMessage[res.statusCode] !== undefined) {
    badgeData.text[1] = errorMessage[res.statusCode];
    badgeData.colorscheme = 'lightgrey';
    return true;
  } else if (res.statusCode !== 200) {
    badgeData.text[1] = 'invalid';
    badgeData.colorscheme = 'lightgrey';
    return true;
  } else {
    return false;
  }
};

//Usage:
checkErrorResponse(badgeData, err, res, {403: 'repo not found'}); // 403: repo not found
checkErrorResponse(badgeData, err, res, 'repo not found'); // 404: repo not found
checkErrorResponse(badgeData, err, res, {403: 'repo not found', 404: 'repo not found'}); // 403, 404: repo not found
```

That way if services return a strange error code, we can still set a relevant error message.

Reply:
**inaccessible** would be a nice way to convey a permission problem. For our badges, thoguh, it usually means the service can't be reached.

Maybe **not authorized**?

Reply:
Yes, "not authorized" may be good alternatives for 403 cases. "access denied" is a good alternative as well. I'm leaning towards a separate PR though, as this has not got much to do with "undefined" in GitHub badges.